modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -78,14 +91,14 @@ If MS2-Randomize were set to `true`, the payload in the response from the mockin
 }
 ----
 
-MS2-Status-Code:: Specifies the non-error code and the error code that the mocking service can choose from for the return message. Separate the two status codes with a comma. For example, you might want the mocking service to return a 201 message, if there is no error, or a 401 status code, if there is an error; in this case, you would specify `201,401` for the value.
-+
-If you use this header and you do not provide a non-error status code, the mocking service throws an error if the API does not return an error. Also, if you use this header and you do not provide an error status code, the mocking service throws an error if the API returns an error.
+MS2-Status-Code:: Specifies the status code or codes that the mocking service can choose from for the return message. This header behaves as a filter on the list of status codes that are defined for a method. For example, suppose a method has several non-error status codes defined; if you wanted the mocking service to respond with only one of those status codes, you would specify it with this header.
 +
-If you do not use this header, the mocking service behaves as follows:
+[IMPORTANT]
+====
+If you use this header together with MS2-Error-Rate, you must specify both a non-error status code and an error status code. Separate the two status codes with a comma. For example, you might want the mocking service to return a 201 message, if there is no error, or a 401 status code, if there is an error; in this case, you would specify `201,401` for the value.
 +
-* If the API returns a non-error status code, the mocking service returns the first non-error status code that is listed for the method.
-* If the API returns an error status code, the mocking service returns the first error status code that is listed for the method.
+If you use this header and you do not provide a non-error status code, the mocking service throws an error if, based on the probability in MS2-Error-Rate, the mocking service does not attempt to respond with an error. Also, if you use this header and you do not provide an error status code, the mocking service throws an error if the mocking service does attempt to respond with an error.
```

Comment:
I think this paragraph adds some confusion, I'd just remove it because it is covered by the previous paragraph
> If you use this header and you do not provide a non-error status code, the mocking service throws an error if, based on the probability in MS2-Error-Rate, the mocking service does not attempt to respond with an error. Also, if you use this header and you do not provide an error status code, the mocking service throws an error if the mocking service does attempt to respond with an error.

Reply:
Removed.